### PR TITLE
revise doc guide toc

### PR DESCRIPTION
**Pull request by Bradamant3**
Work in progress. Not sure about how good to merge yet, but getting something up there to help folks visualize https://github.com/writethedocs/www/issues/527

### CODE REVIEW

**Comment by plaindocs** [comment]
What is the status on this? I can take a deeper look, but probably fine to 'just merge' TM?

**Comment by Bradamant3** [comment]
I guess we can go ahead and merge. I'd like to get some editorial content in there about the documentation principles and style guides contributions, but I need to think -- and maybe discuss with some of y'all -- the best way to proceed. We don't have a DCO or any sort of contributor agreement, do we? Safe to assume that any and all content edits are OK?

**Comment by plaindocs** [comment]
Oops, dropped the ball here. @Bradamant3 I'll leave this open till you're online and merge if that sounds good. Apologies.

**Comment by ericholscher** [comment]
This is now conflicting. Seems valuable, so we should get it fixed up and merge it.

**Comment by Bradamant3** [comment]
@plaindocs @ericholscher conflicts fixed

File: docs/guide/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Markup
+------
+
+Some of the current files in the guide are written in Markdown, and some are written in reStructured Text (rST). Either one is acceptable, although we'd prefer rST.
+
+# Community
+
+For help, questions, or discussion, the Write the Docs community is available here:
+
+* [Slack](https://writethedocs.slack.com/)
```
Comment (ericholscher) [inline docs/guide/contributing.rst]: This is still in markdown

File: docs/guide/index.rst [hunk 2]
```diff
@@ -2,65 +2,51 @@
...
 **This is a living, breathing guide.**
 If you'd like to contribute,
 you can read our :doc:`contributing <contributing>` documentation.
 Also feel free to send us any additions in any format to support@writethedocs.org.
 
-.. note:: There are a number of topics that aren't written yet.
-          That means that you can write them!
-          Please submit your thoughts and ideas,
-          and you can contribute to making this guide better.
+Take a look also at the `guidelines for contributing to the guide <contributing>`_. If you have any questions about contributing,
```
Comment (ericholscher) [inline docs/guide/index.rst]: This should use `:doc:`, not a relative link, so that it works across builders.

**Review by ericholscher — COMMENTED** [review]
Looks good with a few nits.

File: docs/guide/contributing.rst (hunk 1 above)
Comment (plaindocs) [inline docs/guide/contributing.rst]:
```suggestion
- `Slack <http://slack.writethedocs.com/>`_
```

File: docs/guide/contributing.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Write the Docs forum](http://forum.writethedocs.org/)
```
Comment (plaindocs) [inline docs/guide/contributing.rst]:
```suggestion
- `Forum <http://forum.writethedocs.org/>`_
```

File: docs/guide/index.rst (hunk 2 above)
Comment (plaindocs) [inline docs/guide/index.rst]:
```suggestion
Take a look also at the :doc:`guidelines for contributing to the guide <contributing>`. If you have any questions about contributing, 
```

File: docs/guide/contributing.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+For small changes, you can submit issues, which you can do in the GitHub interface, but for anything larger you'll probably want to learn how to fork and branch, how to work locally (although you can do most things in the GitHub interface in your branch), and how to keep your fork in sync with the main repository.
+
+If you really don't want to deal with GitHub at all, we'll take attachments or inline text emailed to guide@writethedocs.org. But we'd like to encourage you to learn some of the basic tools that we use.
+
+Markup
+------
+
+Some of the current files in the guide are written in Markdown, and some are written in reStructured Text (rST). Either one is acceptable, although we'd prefer rST.
+
+# Community
```
Comment (plaindocs) [inline docs/guide/contributing.rst]:
```suggestion
Community
----------------
```

File: docs/guide/contributing.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+Keep your tone friendly and encouraging. It's a good way to write docs in general, so practice with docs about docs!
+
+Link liberally to resources that you find useful, but make sure to provide appropriate attribution. And make sure to add something about how or why you find them useful.
+
+For anyone new to GitHub and Git
+--------------------------------
+
+If you've never worked with Git and GitHub before, try these resources:
+
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) is a great resource to help you get started.
```
Comment (plaindocs) [inline docs/guide/contributing.rst]:
```suggestion
`How to Contribute to an Open Source Project on GitHub <https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github>`_ is a great resource to help you get started.
```

File: docs/guide/contributing.rst [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+# Community
+
+For help, questions, or discussion, the Write the Docs community is available here:
+
+* [Slack](https://writethedocs.slack.com/)
+
+* [Write the Docs forum](http://forum.writethedocs.org/)
+
+You can also check out [the main website](http://www.writethedocs.org/meetups/) for a list of local meetups.
```
Comment (plaindocs) [inline docs/guide/contributing.rst]:
```suggestion
You can also check out our `local meetups <http://www.writethedocs.org/meetups>`_.
```

**Review by plaindocs — CHANGES_REQUESTED** [review]
:+1: think I caught most of these

**Comment by Bradamant3** [comment]
perhaps GH's beta `commit suggestion` feature is not quite so cool? At any rate, seems travis is not happy on some of those commits ... or are they "just" waiting on a new build? Confused, but grateful to @plaindocs and amused also at my inability to stay in .rst after too long in .md.

**Review by plaindocs — APPROVED** [review]

**Pull request merged**